lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -880,8 +879,7 @@ balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
 -- much ada.
 selectAssets
     :: forall era changeState
-     . Cardano.IsCardanoEra era
-    => RecentEra era
+     . RecentEra era
```

Comment:
I'd like to request 4-space indents here!

I would suggest:
```hs
selectAssets
    :: forall era changeState. RecentEra era
    -> ProtocolParameters era
    -> UTxOAssumptions
```
Or, if you're **_really_** determined to have `RecentEra` line up with everything below (😄), then:
```hs
selectAssets
    :: forall era changeState. ()
    => RecentEra era
    -> ProtocolParameters era
    -> UTxOAssumptions
```